plugins/netscaler-k8s.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: netscaler-k8s
+spec:
+  shortDescription: A Kubernetes plugin for inspecting Ingress Controller and associated NetScaler deployments
+  description: |
+    The official kubectl plugin for netscaler-k8s.
+  version: v1.0.0
+  homepage: https://github.com/netscaler/modern-apps-toolkit/tree/main/netscaler-k8s-plugin#readme
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: "https://github.com/netscaler/modern-apps-toolkit/releases/download/v1.0.0-netscaler-k8s-plugin/netscaler-k8s-plugin_v1.0.0-netscaler-k8s-plugin_Darwin_x86_64.tar.gz"
+    sha256: 0d374bd0073c067055369b842ab56a2b96af16cc71513e92a846b71eec1ec93e
+    bin: ./kubectl-netscaler_k8s
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this is redundant, you can drop `files` property altogether.

Reply:
Fixed in latest commit